phase1-coordinator/src/objects/round.rs

```diff
@@ -524,12 +570,7 @@ impl Round {
         // Check that the participant is authorized to acquire the lock
         // associated with the given chunk ID for the current round,
         // and fetch the appropriate contribution locator.
-        let (
-            previous_contribution_locator,
-            current_contribution_locator,
-            next_contribution_locator,
-            contribution_file_signature_locator,
-        ) = match participant {
+        let locked_locators = match participant {
```

Comment:
I was hoping you would go farther to make it a separate function. We can still do it later if you like the idea

Reply:
I think I was saving that kind of change for when we make each participant type its own type in #245